_internals/tds-protocol.md

```diff
@@ -85,8 +85,27 @@ Once we are over the <code>PRELOGIN</code> phase, we are ready for
 <code>LOGIN7</code> which defines the authentication rules for use 
 between client and server.
 
-
-### Defining authentication rules.
+### SSL/TLS support for TDS connections
+
+Babelfish uses native PostgreSQL support for using SSL connections to encrypt client/server communications. See PostgresSQL [documentation](https://www.postgresql.org/docs/13/ssl-tcp.html#SSL-SETUP) for details about configuring server-side SSL functionality.
+With SSL support compiled in, the PostgreSQL server can be started with SSL enabled by setting the parameter <code>ssl</code> to <code>on</code> in postgresql.conf. The server will listen for both normal and SSL connections on the same TCP port, and will negotiate with any connecting client on whether to use SSL. By default, a client can decide whether to use SSL connections or not. If <code>babelfishpg_tds.tds_ssl_encrypt</code> is set to <code>true</code>, end-to-end encryption is enforced for all connections. In that case, if a client requests for a non-encrypted connection, the connection is rejected.
+
+The table following shows how Babelfish behaves for each combination.
```

Comment:
How about:

The following table shows how Babelfish interprets SSL settings when a client connects: